Tests/PackageLoadingTests/PD4_2LoadingTests.swift

```diff
@@ -500,6 +500,11 @@ class PackageDescription4_2LoadingTests: PackageDescriptionLoadingTests {
     }
 
     func testCacheInvalidationOnEnv() throws {
+        #if os(Linux)
```

Comment:
Could we conditionalize on `aarch64` as well, so we still get the normal Linux testing?

Reply:
Actually I get `Unknown architecture for build configuration 'arch'` when trying that locally, so it might not be possible.

Reply:
Yah, same here. So I think we merge this as-is for now.